feed/index.html

```diff
@@ -12,10 +12,10 @@
     <title>Feed page</title>
   </head>
 
-  <body>
+  <body class="bg-primary">
     <nav class="navbar navbar-expand-lg bg-secondary shadow">
       <div class="container">
-        <a class="navbar-brand" href="#">
+        <a class="navbar-brand" href="../profile/index.html">
```

Comment:
```suggestion
        <a class="navbar-brand" href="/profile/index.html">
```

This should work, `/` means "from the root of your project"

Reply:
```suggestion
        <a class="navbar-brand" href="/profile/">
```

Even better :)

Reply:
Ooh, nice! It worked